src/flutter-for-ios.md

```diff
@@ -1938,7 +1938,7 @@ class _SampleAppPageState extends State<SampleAppPage> {
     String emailRegexp =
         r'^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$';
 
-    RegExp regExp = RegExp(p);
+    RegExp regExp = RegExp(emailRegexp);
 
     return regExp.hasMatch(em);
```

Comment:
Good coding practice would have this `em` renamed to something like `emailString`. `em` isn't very descriptive...